06_create_cluster.sh

```diff
@@ -22,4 +22,8 @@ for name in dnsmasq ironic-inspector ; do
     sudo podman ps | grep -w "$name$" && sudo podman stop $name
 done
 
+# Configure storage for the image registry
+oc --config ${KUBECONFIG} patch configs.imageregistry.operator.openshift.io \
```

Comment:
CI is complaining about this:

```
+(./06_create_cluster.sh:26): main(): oc --config /root/dev-scripts/ocp/ostest/auth/kubeconfig patch configs.imageregistry.operator.openshift.io cluster --type merge --patch '{"spec":{"storage":{"emptyDir":{}},"managementState":"Managed"}}'
Error: unknown flag: --config
```

Not sure why: --config should be the right option.  But, KUBECONFIG env var should already be set anyway.

Reply:
I guess it's `--kubeconfig`. I removed the flag though.